code/modules/recycling/belt-placer.dm

```diff
@@ -0,0 +1,42 @@
+/obj/item/storage/conveyor //Stores conveyor belts, click floor to make belt, use a conveyor switch on this to link all belts to that lever.
+	name = "conveyor belt placer"
+	desc = "This device facilitates the rapid deployment of conveyor belts."
+	icon_state = "belt_placer"
+	w_class = WEIGHT_CLASS_BULKY //Because belts are large things, you know?
+	can_hold = list(/obj/item/conveyor_construct)
+	max_w_class = WEIGHT_CLASS_BULKY
+	max_combined_w_class = 28 //7 belts
+	allow_quick_gather = TRUE
+	allow_quick_empty = TRUE
+	display_contents_with_number = TRUE
+	use_to_pickup = TRUE
+	origin_tech = "engineering=1"
+
+/obj/item/storage/conveyor/bluespace
```

Comment:
I'd rather not have more infinite storage items, please.

The ore is bag is bad enough.

Reply:
Oh, you might be right. I'll limit the number they can store, would 50 be too much?

Reply:
50 is how much a plant bag can carry of plant stuff. It should be fine there